components/features/SimpleStepsCreate.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <section class="container py-5 mt-3">
+    <h2 class="section__heading text-center">
+      {{ heading }}
+    </h2>
+
+    <div class="py-5 row">
+      <div v-for="(step,idx) in steps" :key="step?.title" :class="columnClass">
+        <div
+          class="d-flex flex-column-reverse align-items-center justify-content-center pe-md-4"
+        >
+          <div class="mt-5">
+            <span class="step__heading">Step {{idx+1}}</span>
+            <h3 class="step__section-heading">{{ step?.title }}</h3>
+            <p class="step__section-description">
+              {{ step?.description }}
+            </p>
+          </div>
+          <div class="d-flex flex-column flex-lg-row">
+            <nuxt-img
+              :src="step?.imgSrc"
+              class="img-fluid my-auto"
+              alt="Signup with a free account"
```

Comment:
@partha6686 the alt would be different for different images. You can get this from the props itself

Reply:
I think we can add the title as the alt.`:alt="step?.title"`

Reply:
we can, but it's not a good practice. Alt is generally the textual representation of an image when the image is not visible.

Reply:
yes, got it.